### PR TITLE
Changes on custom_load

### PR DESCRIPTION
**Pull request by Helm-droid**
1. Changes on _saveto function: This is to fix the bug after the result is reloaded again. The initial error was found due to the 'tolist' function as we are required to have them after inference is made to convert arrays to list. However, if we loaded the results from the .pkl files, we are not required to do it again.
2. Removing '.pop' of 'classification' in the load_data function and replace them in _Visualizer function instead
3. Changes to SavetoLas function: Separated the if statement of the first one from the other two since if directory is set to None, it wont go into the elseif option to create a directory for itself.
4. Now the files will be saved in the correct directory rather than the root project (also changes in _saveto function)

### CODE REVIEW

File: utils/custom_load.py [hunk 1]
```diff
@@ -473,13 +498,13 @@ def JsonVisualizer(self,dir_path = None):
...
-    def SavetoLas(self, results, dir_path = "results/"): 
-
+    def SavetoLas(self, results, dir_path = None): 
+                
         print("Saving point cloud in LAS format with segementation results...")
         if dir_path is None:
-            dir_path = os.path.join(self.dir, "/las")
-            print(f"Folder {dir_path} created in current directory.")
-        elif not os.path.exists(dir_path):
+            dir_path = os.path.join(self.dir, "las")
```
Comment (tniveej) [inline utils/custom_load.py:505]: need to have forward slash after las? ` dir_path = os.path.join(self.dir, "las/")` ?

**Review by tniveej — COMMENTED** [review]
If comment does not apply, just merget

File: utils/custom_load.py (hunk 1 above)
Comment (Helm-droid) [reply utils/custom_load.py:505]: no need. I tried to print them, the os.path.join will automatically provide a forward slash. You can check the object instances at the very start. Try and print them out to see them as well.

Comment (tniveej) [reply utils/custom_load.py:505]: Got it. I will merget.

**Pull request merged**